p2p/CONFIG.md

```diff
@@ -214,6 +214,15 @@ Sets the upper bound on the number of peers we keep in a GossipSub topic mesh.
 | ---                    | ---                 | ---                  |
 | `12`                   | `12`                | `12`                 |
 
+### Gossipsub: Dout
+
+Sets the number of outbound peers we keep in a GossipSub topic mesh.
+
+| Default Value (libp2p)  | Default Value (SSV) | Default Value (ETH2) |
+|-------------------------|---------------------|----------------------|
+| `2`                     | `4`                 | `2`                  |
```

Comment:
add explanation to why its different from default and eth2

Reply:
done